buildSrc/src/main/java/Config.kt

```diff
@@ -57,6 +57,8 @@ object Config {
         val springAop = "org.springframework:spring-aop"
         val aspectj = "org.aspectj:aspectjweaver"
         val servletApi = "javax.servlet:javax.servlet-api"
+
+        val apacheHttpClient = "org.apache.httpcomponents.client5:httpclient5:5.0.3"
```

Comment:
1st stable version of this lib v5 appeared in Feb, 2020, do you think that the majority of Apps have been upgraded already?

Reply:
There is no package collision between version 5 and 4 so even if they did not, it means just an extra dependency. We still can add transport based on 4.x if there will be such request.